lineman/app/components/base.scss

```diff
@@ -6,7 +6,7 @@ body {
 }
 
 h1, h2, h3, h4{
-  @include fontReset;
+  @include fontReset; /* Y U SO RESET? */
```

Comment:
Headings are necessary to make the page as accessible as possible, but we don't want the same styles between the headings on different pages.